test/integration/default/controls/config_spec.rb

```diff
@@ -23,7 +27,7 @@
     its('content') { should include 'user' }
   end
 
-  describe command('ls -l /etc/openvpn/myserver1.conf') do
+  describe command("ls -l #{cfgfile}") do
```

Comment:
Is this test used to verify owner and group of this cfgfile?
Then you can do it directly with Inspec resources, like https://github.com/saltstack-formulas/template-formula/blob/develop/test/integration/default/controls/config_spec.rb#L6